css/logo-nav.css

```diff
@@ -893,11 +892,14 @@ a {
         margin-top: 20px;
     }
 }
-@media (max-width: 768px) {
+@media (max-width: 767px) {
```

Comment:
Did their logo say "Sponsored by NumFocus" or did we add that. I am not
sure it is quite correct to say "Sponsored by" A more accurate way of
phrasing it would be "A NumFOCUS Project"

On Sat, Apr 16, 2016 at 4:33 AM, Matthias Bussonnier <
notifications@github.com> wrote:

> In css/logo-nav.css
> <https://github.com/jupyter/jupyter.github.io/pull/84#discussion_r59966292>
> :
>
> > @@ -893,11 +892,14 @@ a {
> >          margin-top: 20px;
> >      }
> >  }
> > -@media (max-width: 768px) {
> > +@media (max-width: 767px) {
>
> Why do you change to 767 ? We already have a media query at 768, changing
> to 767 make 2 transition both at 767 and 768 which looks weird.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter.github.io/pull/84/files/c09a9d89e396afbba52c793deadb67cb105d3ac1#r59966292>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com